packages/flutter_driver/lib/src/common/text_input_action.dart

```diff
@@ -0,0 +1,189 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'enum_util.dart';
+import 'message.dart';
+
+EnumIndex<TextInputAction> _textInputActionIndex =
+    EnumIndex<TextInputAction>(TextInputAction.values);
+
+/// A Flutter Driver command that send a text input action.
+class SendTextInputAction extends Command {
+  /// Creates a command that enters text into the currently focused widget.
+  const SendTextInputAction(this.textInputAction, {super.timeout});
+
+  /// Deserializes this command from the value generated by [serialize].
+  SendTextInputAction.deserialize(super.json)
+      : textInputAction =
+            _textInputActionIndex.lookupBySimpleName(json['action']!),
+        super.deserialize();
+
+  /// The [TextInputAction]
+  final TextInputAction textInputAction;
+
+  @override
+  String get kind => 'send_text_input_action';
+
+  @override
+  Map<String, String> serialize() => super.serialize()
+    ..addAll(<String, String>{
+      'action': _textInputActionIndex.toSimpleName(textInputAction),
+    });
+}
+
+/// An action the user has requested the text input control to perform.
+///
+/// This class is identical to [TextInputAction](https://api.flutter.dev/flutter/services/TextInputAction.html).
+/// This class is cloned from `TextInputAction` and must be kept in sync. The cloning is needed
+/// because importing is not allowed directly.
```

Comment:
```suggestion
// This class is identical to [TextInputAction](https://api.flutter.dev/flutter/services/TextInputAction.html).
// This class is cloned from `TextInputAction` and must be kept in sync. The cloning is needed
// because importing is not allowed directly.
```

Reply:
Fixed, thanks.